google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -918,4 +918,38 @@ public void testCustomAsyncExecutorProvider() {
             .build();
     assertSame(service, options.getAsyncExecutorProvider().getExecutor());
   }
+
+  @Test
+  public void testDefaultNumChannelsWithGrpcGcpExtensionEnabled() {
+    SpannerOptions options =
+        SpannerOptions.newBuilder().setProjectId("test-project").enableGrpcGcpExtension().build();
+
+    assertEquals(options.getNumChannels(), SpannerOptions.GRPC_GCP_ENABLED_DEFAULT_CHANNELS);
+  }
+
+  @Test
+  public void testNumChannelsWithGrpcGcpExtensionEnabled() {
+    // Set number of channels explicitly, before enabling gRPC-GCP channel pool in SpannerOptions
+    // builder.
+    int numChannels = 5;
```

Comment:
Anything special about 5?

Reply:
There is nothing special about 5. Just chose it randomly to check functionality of enabling number of channels explicitly.